lua/modules/linear_bolzmann_solvers/lbs_solver/lbs_sources.cc

```diff
@@ -55,16 +55,16 @@ RegisterLuaFunctionInNamespace(AddDistributedSource, lbs, AddDistributedSource);
 int
 AddDistributedSource(lua_State* L)
 {
-  const std::string fname = "lbs.AddDistributedSource";
+  const std::string fname = "lbs.AddVolumetricSource";
```

Comment:
Should `AddDistributedSource` (i.e. the function name) be changed to `AddVolumetricSource`. Also, the same name change for the lua function? This seems to be changing the `fname` which is what the user would see in case of an error.